### ISSUE TITLE
Mod for Game Updating while time stopped with TimeSpeed Mod

### ISSUE THREAD

**Issue by AnnyAnnye** [StardewModders/mod-ideas#469]
Hi! I need some mod or tweak which makes worm spots and foraging/shells appears, crops and trees grow, machines work, trees drop acorn/pine/maple seeds, can gift townies and/or anything else that need updates and usually happen after day change, but I need the updates happening after real-life time spent in-game, as I suffer from anxiety and cant play the game with normal speed, please.

**Comment by KarBoy2314PL** [comment]
there are mods to modify the clock and calendar as well as one for modifying worm spots, but im not sure what do you exactly want

**Comment by kdau** [comment]
I saw the history of this request on Discord. The issue for this player is that, when using existing mods that pause the clock, things that update on the ten-minute or daily intervals don't happen. They would like an option to have those things happen as if the clock were progressing, even though the time does not change and days do not pass.

**Comment by KarBoy2314PL** [comment]
>and days do not pass
this might be very hard, considering some events are locked to a specific day on the calendar

**Comment by kdau** [comment]
Definitely, this is a "very hard" SMAPI mod just for the number of Harmony patches involved to rework significant chunks of update logic. I suppose the alternate option would be to have time and the day cycle pass as usual, but suppress sleep and the nighttime visual changes and allow saving and shipping bin processing to occur at arbitrarily chosen times, a bit like Save Anywhere plus that no-sleep mod that stopped working.

**Comment by tiakall** [comment]
This is requesting a feature to Timespeed, which should be suggested to the mod author instead.
